### PR TITLE
docs: updated README.md file to include latest k8s namespaces

### PR DESCRIPTION
**Pull request by frank-dodion**
## Ticket Number
AS-1312

## Description of change
Updated README.md file to include latest k8s namespaces

### CODE REVIEW

**Review by rcmsoftwaredevelopment — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -232,14 +232,16 @@ az login
...
+# pins-uks-k8s-preprod
 # pins-uks-k8s-prod
 ```
 
 ```shell script
 az aks list -o tsv --query '[].name'
 
-# pins-uks-k8s-1234-dev
-# pins-uks-k8s-1234-prod
+# pins-uks-k8s-5883-dev
```
Comment (MrSimonEmms) [inline README.md:242]: These may change - might be worth putting a note to that effect

**Review by MrSimonEmms — CHANGES_REQUESTED** [review]

File: README.md [hunk 2]
```diff
@@ -248,16 +250,36 @@ The name format will include the environment you're after at the end of the name
...
 Kubernetes cluster.
 
 ```shell script
-kubectl logs -f -n app-prod app-form-web-app-1111111111-aaaaa
+kubectl logs -f -n app-dev app-form-web-app-1111111111-aaaaa
+```
+
+The available namespaces can be listed with the command:
+```shell script
+kubectl get namespace
```
Comment (MrSimonEmms) [inline README.md:267]: Does this command work for you now? Or is that in on the assumption I get it working from my end?

File: README.md (hunk 1 above)
Comment (frank-dodion) [reply README.md:242]: OK.

File: README.md (hunk 2 above)
Comment (frank-dodion) [reply README.md:267]: The command is correct. Whether it works will depend on individual permissions. It was not working for me when I checked last.

**Review by MrSimonEmms — APPROVED** [review]
Good work @frank-dodion

**Pull request merged**